projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/interop/LLVMTckTest.java

```diff
@@ -0,0 +1,410 @@
+        super.testNullInCompoundObject();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testFortyTwoWithCompoundObject() throws Exception {
+        super.testFortyTwoWithCompoundObject();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPlusWithIntsOnCompoundObject() throws Exception {
+        super.testPlusWithIntsOnCompoundObject();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteBooleanValue() throws Exception {
+        super.readWriteBooleanValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteByteValue() throws Exception {
+        super.readWriteByteValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteShortValue() throws Exception {
+        super.readWriteShortValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteCharValue() throws Exception {
+        super.readWriteCharValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteIntValue() throws Exception {
+        super.readWriteIntValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteFloatValue() throws Exception {
+        super.readWriteFloatValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void readWriteDoubleValue() throws Exception {
+        super.readWriteDoubleValue();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPrimitiveReturnTypeByte() throws Exception {
+        super.testPrimitiveReturnTypeByte();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPrimitiveReturnTypeShort() throws Exception {
+        super.testPrimitiveReturnTypeShort();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPrimitiveReturnTypeInt() throws Exception {
+        super.testPrimitiveReturnTypeInt();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPrimitiveReturnTypeLong() throws Exception {
+        super.testPrimitiveReturnTypeLong();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPrimitiveReturnTypeFloat() throws Exception {
+        super.testPrimitiveReturnTypeFloat();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testPrimitiveReturnTypeDouble() throws Exception {
+        super.testPrimitiveReturnTypeDouble();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testInvalidTestMethod() throws Exception {
+        super.testInvalidTestMethod();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testEvaluateSource() throws Exception {
+        super.testEvaluateSource();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void timeOutTest() throws Exception {
+        super.timeOutTest();
+    }
+
+    @Override
+    @Ignore
+    @Test
+    public void testCoExistanceOfMultipleLanguageInstances() throws Exception {
+        super.testCoExistanceOfMultipleLanguageInstances();
+    }
```

Comment:
Yes, there are. Either they check something with structs, or they check something with function passing, which is not yet implemented.